### PR TITLE
 Allow alternate service types without enabling gke #4388 

### PR DESCRIPTION
**Pull request by joesteffee**
Deploying in EKS with an ALB ingress requires either type=LoadBalancer or type=NodePort. The only way to enable NodePort today is by enabling GKE which enables some other stuff we don't want. This allows the user to set a custom service type

Enable use of custom paths for ingress (alb ingress controller requires /* instead of /)

### CODE REVIEW

File: dojo/fixtures/benchmark_requirement.json [hunk 1]
```diff
@@ -7492,7 +7492,7 @@
     "pk": 418,
     "fields": {
       "category": 27,
-      "objective_number": "1.6.1",
+      "objective_number": "1.6.2",
```
Comment (madchap) [inline dojo/fixtures/benchmark_requirement.json:7495]: This file is the wrong file, I guess you were looking for Chart.yaml instead to bump the chart number.
It seems the linting didn't catch it, so I'll have to look again. However, you'd want to bump the Chart version to 1.6.3.

**Review by madchap — CHANGES_REQUESTED** [review]

File: dojo/fixtures/benchmark_requirement.json (hunk 1 above)
Comment (joesteffee) [reply dojo/fixtures/benchmark_requirement.json]: fixed

**Review by madchap — APPROVED** [review]

**Review by mtesauro — APPROVED** [review]
Approved

**Comment by mtesauro** [comment]
@madchap Not sure why those k8s tests are getting cancelled TBH

**Pull request merged**